lib/Models/Catalog/Ows/WebMapServiceCapabilitiesStratum.ts

```diff
@@ -852,10 +856,23 @@ export default class WebMapServiceCapabilitiesStratum extends LoadableStratum(
 
     if (formatsArray.includes("application/json"))
       return { format: "application/json", type: "json" };
+    if (formatsArray.includes("application/geo+json"))
+      return { format: "application/geo+json", type: "json" };
+    if (formatsArray.includes("application/vnd.geo+json"))
+      return { format: "application/vnd.geo+json", type: "json" };
+
+    // Special case for Esri WMS, use XML before HTML
+    // as HTML includes <table> with rowbg that is hard to read
+    if (this.isEsri && formatsArray.includes("text/xml")) {
```

Comment:
The test of `this.isEsri` is redundant to the test at line 873.

Reply:
The test at 873 only applies for non-esri services - I haven't included `!this.isEsri` in the condition because it is superfluous 

The logic is this

- If service is ESRI, and supports use XML, use that
- For all services, if supports HTML, use that
- For all services, if supports GML, use that
- For all (non-ESRI) services, if supports XML, use that
- ...

This means for ESRI service, XML will be used before HTML and GML. 
But for non ESRI services, then HTML and GML will be used before XML

I have added a comment to clarify this